### DISCUSSION TITLE
libplist

### DISCUSSION THREAD

**Discussion by rdgnbrs** [Homebrew/discussions#4683] [Casks]
Hello,

I need libplist 2.3.0 in order to install MVT for Apple M1 MacBook Air.

The version in the homebrew repistory is libpbl 2.2.0.

Can you update libplist package from 2.2.0 to 2.3.0.

Kind regards

**Comment by SMillerDev** [comment]
No https://github.com/Homebrew/homebrew-core/pull/128974
